pkg/bundle/resources.go

```diff
@@ -306,13 +306,19 @@ func readContent(ctx context.Context, base, name string, auth Auth) (map[string]
 		return nil, err
 	}
 
+	// copy getter.Getters before changing
+	getters := map[string]getter.Getter{}
+	for k, v := range getter.Getters {
+		getters[k] = v
+	}
+
 	c := getter.Client{
 		Ctx:     ctx,
 		Src:     name,
 		Dst:     temp,
 		Pwd:     base,
 		Mode:    getter.ClientModeDir,
-		Getters: getter.Getters,
+		Getters: getters,
```

Comment:
Further down in the code, the original `getter.Getters` is being updated. That should use the new `getters` instead. See...

https://github.com/rancher/fleet/blob/bd0fd515a2823c68b0ce0e5eab71558811fadd0c/pkg/bundle/resources.go#L350-L351

Can you please update all the uses of `getter.Getters`.

Reply:
this is `c.Getters` that already refer to copied getters map, but I'll change it anyway for the sake of readability